### ISSUE TITLE
About the processing functional differences between the two implementations 

### ISSUE THREAD

**Issue by Francyfx** [skulpt/skulpt#797]
If I want to use java/python processing function,how do that
when I run 
size(400,400)
print(width)
it show 100 default but 400.

**Comment by albertjan** [comment]
@Francyfx try putting the code in a setup function and calling run at the end.

```
def setup():
    size(400,400)
    print(environment.width)

run()
```

**Comment by Francyfx** [comment]
Here is a table that lists the functional differences between the two implementations.
https://github.com/skulpt/skulpt/wiki/Skulpt-Processing.
If I still want to use python/java processing instead of skulpt processing grammer like 'MouseX', 'width' and so on.

**Comment by albertjan** [comment]
Yeah the problem is though that mod.width will get set when the module is loaded and won't get updated again afterwards.

**Comment by albertjan** [comment]
@Francyfx Have a look at https://github.com/trinketapp/processing.sk that does support width and mouseX being upto date. But that also requires a change to skulpt. See these lines in misceval.js https://github.com/trinketapp/skulpt/blob/master/src/misceval.js#L688-L704

**Comment by Francyfx** [comment]
I have try it as you method ,but it dosen`t work.
Is there a html example I can see in  browser.
There isn`t a example in https://github.com/trinketapp/processing.sk

**Comment by Francyfx** [comment]
Well,thanks.
here is my processing code with skulpt.
In trinket I couldn`t find some code like this,it is hard to read.
![image](https://user-images.githubusercontent.com/28282922/46843483-97b24080-ce05-11e8-89cc-f113afafe519.png)
![image](https://user-images.githubusercontent.com/28282922/46843492-a39e0280-ce05-11e8-98e0-7eecdb0cb0ba.png)
